bbp/modules/default.nix

```diff
@@ -682,22 +682,47 @@ let
             moduleFilePrefix = "nix/nse";
 
             description = "neuroM module generated by nix";
-            packages = [
-                            pkgs.neurom
-                            ## include all dependencies in the env
-                       ] ++ pkgs.neurom.pythonDeps;
+            packages = with pkgs.pythonPackages; (
+                getPyModRec [ pkgs.neurom ]
+            );
 
             conflicts = conflicts-modules;
+            dependencies = [ python27-full ];
```

Comment:
I saw that in some other Python modules here.
Should it be considered a practice to stick to for Python 2 / 3 library modules?

Reply:
Yes, this is a good idea. to requires the proper Python executable to prevent misuse.
In your case, as module provide all required packages, dependencies should be python27-light IMHO.